Scripts/parameters/assignment.py

```diff
@@ -277,9 +277,9 @@
     "@trailer_truck": "LINK",
     "@truck": "LINK",
     "@van": "LINK", 
-    "@transit_vol": "TRANSIT_SEGMENT",
-    "@transit_boa": "TRANSIT_SEGMENT",
-    "@transit_trb": "TRANSIT_SEGMENT",
+    "@transit_vol": "LINK",
+    "@transit_boa": "NODE",
+    "@transit_trb": "NODE",
```

Comment:
Do you have any clue what the impact on the required words for extra attributes will be?

Reply:
I'm not sure how words for extra attributes are calculated, but as these extra attributes were initialized before as segment attributes it could be less than before (number of segments > number of links / nodes)?